server/routes_create_test.go

```diff
@@ -21,9 +22,14 @@ import (
 
 var stream bool = false
 
-func createBinFile(t *testing.T, kv map[string]any, ti []llm.Tensor) string {
+func createBinFile(t *testing.T, kv map[string]any, ti []llm.Tensor) (string, string) {
```

Comment:
This now returns both the filename which was created for the bin file, along with its digest.